addons/binding/org.openhab.binding.netatmo/ESH-INF/thing/welcomeCamera.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="netatmo"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+      
+    <thing-type id="NAWelcomeCamera">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="netatmoapi" />
+        </supported-bridge-type-refs>
+        
+        <label>Welcome Camera</label>
+        <description>This represents a welcome camera at home</description>
+
+        <channels>
+            <channel id="welcomeCameraId" typeId="id"></channel>
+        	<channel id="welcomeCameraType" typeId="type"></channel>
+        	<channel id="welcomeCameraStatus" typeId="status"></channel>
+        	<channel id="welcomeCameraSdStatus" typeId="sd_status"></channel>
+        	<channel id="welcomeCameraAlimStatus" typeId="alim_status"></channel>
+            <channel id="welcomeCameraName" typeId="name"></channel>
+            <channel id="welcomeCameraVpnUrl" typeId="vpn_url"></channel>
+            <channel id="welcomeCameraIsLocal" typeId="is_locale"></channel>
+            <channel id="welcomeCameraLivePictureUrl" typeId="live_picture_url"></channel>
+            <channel id="welcomeCameraLiveVideoPoorUrl" typeId="live_video_poor_url"></channel>                                   
+            <channel id="welcomeCameraLiveVideoLowUrl" typeId="live_video_low_url"></channel>                                   
+            <channel id="welcomeCameraLiveVideoMediumUrl" typeId="live_video_medium_url"></channel>                                   
+            <channel id="welcomeCameraLiveVideoHighUrl" typeId="live_video_high_url"></channel>                                   
+        </channels>
+              
+        <properties>
+             <property name="actualApp">app_welcomecamera</property>
+        </properties>         
+        
+        <config-description>
+            <parameter name="welcomeHomeId" type="text" required="true" readOnly="true">
```

Comment:
I get the homeID from the netatmo api, don't know if its a good idea to make it possible to manually change it, whats the preferred implementation?